src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSource.java

```diff
@@ -79,36 +79,40 @@ public LookupRuntimeProvider getLookupRuntimeProvider(LookupContext lookupContex
         LookupRow lookupRow = extractLookupRow(lookupContext.getKeys());
 
         DeserializationSchema<RowData> responseSchemaDecoder =
-            decodingFormat.createRuntimeDecoder(lookupContext, physicalRowDataType);
```

Comment:
The reformat of this file seems to not satisfy checkstyle defined for this project.

Reply:
This comment was not addressed.